src/lib/auth/user-resolver.ts

```
@@ -0,0 +1,53 @@
+  }
+  return null;
+};
+
+// Removed duplicate getTenantContext that took NextRequest.
+// The version below that accepts ResolvedUser | null is the one used by route-handler.ts.
+
+// Modify getTenantContext to accept ResolvedUser | null as per usage in route-handler.ts
+export const getTenantContext = async (user: ResolvedUser | null): Promise<TenantContext> => {
+  console.log("Placeholder: getTenantContext called with user:", user);
+  // Actual implementation will be provided by another agent or task.
+  if (user && user.agencyId && process.env.NODE_ENV === 'development') {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)